### ISSUE TITLE
Sleep as Android Error on 2022.3.0b3

### ISSUE THREAD

**Issue by Wibias** [IATkachenko/HA-SleepAsAndroid#38]
### Home Assistant
**Version:2022.3.0b3**
**Installation method** (according to [instructions](https://www.home-assistant.io/installation/)):
[ ] Operating System
[ ✓] Container
[ ] Supervised
[ ] Core
**Installation method:**
[ ✓] HACS
[ ] manual from release
[ ] manual from git
**Integration version:v1.8.0** 
## Debug log
```
2022-02-26 20:11:31 ERROR (MainThread) [homeassistant.components.sensor] Error while setting up sleep_as_android platform for sensor
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/entity_platform.py", line 249, in _async_setup_platform
    await asyncio.shield(task)
  File "/config/custom_components/sleep_as_android/sensor.py", line 48, in async_setup_entry
    await instance.subscribe_root_topic(async_add_entities)
  File "/config/custom_components/sleep_as_android/__init__.py", line 208, in subscribe_root_topic
    self._subscription_state = await subscription.async_subscribe_topics(
TypeError: async_subscribe_topics() takes 2 positional arguments but 3 were given
```

**Comment by IATkachenko** [comment]
Thanks.
Will check it.

**Comment by IATkachenko** [comment]
Fixed in v1.8.1
Because 2022.3 is beta, 1.8.1 is available as beta release too.
